apps/api/src/app/controllers/__tests__/__snapshots__/fetchViewModelById.e2e.spec.ts.snap

```diff
@@ -350,7 +350,7 @@ Object {
     },
   ],
   "data": Object {
-    "audioURL": "https://www.mysound.org/downloads/123.mp3.mp3",
+    "audioURL": "https://www.mysound.org/downloads/Gaas.mp3",
```

Comment:
What's the point of changing this if it's not a live link?

Reply:
The only edit I made was to the `buildTranscribedAudioTestData.ts` file to be able to generate the updated data for the loading up the database using the testData.json file.  In this test, I guess the name of the filename is also taken from `buildTranscribedAudioTestData.ts`.  So this test snapshot gets updated with the new filename.  

I was also wondering about the .mp3.mp3.  When I edited `buildTranscribedAudioTestData.ts` to add the new filenames, I deleted the .mp3 that was a part of each audioFilename and that meant instead of having `<filename>.mp3.mp3`, I got just `<filename>.mp3`.  Is this because `buildFullDigitalAssetURL.ts` adds the `mp3` extension by default?

Reply:
This is a snapshot of a query response, not the test data file itself.